re-frisk/src/re_frisk/ui.cljs

```diff
@@ -56,19 +56,30 @@
     (fn []
       (when-not @ext-opened?
         (let [left (or (utils/normalize-draggable (:x @drag/draggable))
-                       (- js/window.innerWidth 30))]
+                       (- js/window.innerWidth 30))
+              handle-toggle (fn [] 
+                              (when-not (utils/closed? left)
+                                     (swap! db/tool-state assoc :latest-left (- js/window.innerWidth left)))
+                              (swap! drag/draggable assoc :x (- js/window.innerWidth
+                                                                (if (utils/closed? left) @latest-left 30))))
+
+              handle-keydown (fn [e]
+                               (let [input-elements #{"INPUT" "SELECT" "TEXTAREA"}
+                                     input-focused? (contains? input-elements  (.-tagName (.-target e)))]
+                                 (when (and (not input-focused?)
+                                            (= (.-key e) "h")
```

Comment:
could you please elaborate why `ctrl+h` ?

Reply:
ok  i see , re-frame-10x uses this combination, I'm fine with that, btw it would be useful to add settings to hide UI buttons if we have hotkeys, but I can add it later https://github.com/flexsurfer/re-frisk/issues/68